io/disk/multipath_test.py

```diff
@@ -46,9 +46,13 @@ def setUp(self):
         if dist.name == 'Ubuntu':
             pkg_name += "multipath-tools"
             svc_name = "multipath-tools"
-        else:
+        elif dist.name == 'redhat':
```

Comment:
By this implementation, the variables are empty for distros other than these 3.

Suggestion:
```
        if dist.name == 'Ubuntu':
            pkg_name += "multipath-tools"
            svc_name = "multipath-tools"
        elif dist.name == 'SuSE':
            pkg_name += "multipath-tools"
            svc_name = "multipathd"
        else:
            pkg_name += "device-mapper-multipath"
            svc_name = "multipathd"
```